service/src/main/kotlin/fi/espoo/evaka/shared/security/AccessControl.kt

```diff
@@ -331,7 +341,7 @@ WHERE employee_id = :userId
         return result
     }
 
-    private inline fun <reified A, reified I> ActionConfig<A>.hasGlobalPermission(
+    private inline fun <reified A, reified I> ActionConfig<A>.hasPermissionThroughGlobalRole(
```

Comment:
these functions are new to me and I don't know their purpose so I don't know if i broke them